packages/html/src/index.ts

```diff
@@ -3,6 +3,7 @@ export { Interaction, ReadonlyInteraction, isInteraction } from './interaction';
 export { createInteractor } from './create-interactor';
 export { createInspector } from './inspector'
 export { Page } from './page';
+export { Keyboard } from './keyboard';
```

Comment:
Should we move it to a separate package?
```suggestion
export { Keyboard } from '@interactors/keyboard';
```